### DISCUSSION TITLE
Requerimientos

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#73] [Q&A]
Buenas noches, disculpen alguien sabe. 

1.- ¿Cuales son los tipos de demanda que hay?
2.- ¿Datos que se necesitan para recibir una demanda y formato en un word?
3.- Procesos para llevar a cabo las demandas.

Se que se pueden investigar y eso hago pero por si alguien es abogado o sabe al respecto.

**Comment by erykzon** [comment]
Hola ,
Creo que los requerimientos que te refieres es mas bien al formulario , como base debería ser el nombre , teléfono y el correo electrónico . Para que le lleguen las alertas .

**Reply by GloriaCoralCerecedo** [reply]
Si entiendo, yo creo que serian 2. El formulario es para cuando le llegue la notificación al usuario se forme el doc de word. Me imagino que es tipo de demanda, hechos y a lo que te refieres es cuando el usuario crea la cuenta en la plataforma para recibir seguimiento del proceso legal.

**Reply by LuceroLuciano** [reply]
Hola. 
Tengo la misma pregunta, ¿Cuáles son los tipos de demanda que hay?, esto para ponerlo en el formulario así como comentan, yo estoy contemplando datos como: nombre, apellidos, ciudad/país, teléfono, correo electrónico y tipo de demanda, no se, se me ocurre, demanda por robo, divorcio o algún otro tipo, la verdad no se si estoy en lo correcto, pero se me ocurrió porque creo que es importante poner el tipo de demanda que se requiere hacer.  
Disculpen si escribo muy repetitivo, espero me haya dado a entender. 
Saludos.

**Reply by jmlimas** [reply]
Deben ser  penal, mercantil, laborar, Civil etc.  pero faltan mas  datos  que nos  deben  proporcionar  en este  caso nuestro  cliente  el  Abogado que el es el experto en este  tema..

**Reply by GloriaCoralCerecedo** [reply]
Si yo investigue las demandas civiles eso sí solo se dedicará a ellos el despacho de abogados. Igual se puede en general. Y lo que piden para demandas no se si sea general o específico es, son los datos del tributario, del demandado, del demandante, hechos y petición.
Ya de ahí se forma el doc. También investigue el proceso se llama algo así de procesal
